src/StyleManager/Styles.php

```diff
@@ -37,13 +42,15 @@ public function __construct(?array $arrStyles = null)
     /**
      * Return the css class collection of an identifier
      */
-    public function get($identifier, $arrGroups=null): string
+    public function get($identifier, $arrGroups=null, $excludedGroups=null): string
```

Comment:
I think the way to exclude groups is a bit heavy handed. What about another method `->exclude()`?